src/yunohost/backup.py

```diff
@@ -1250,7 +1250,9 @@ def _restore_system(self):
                                '(&(objectClass=permissionYnh)(!(cn=main.mail))(!(cn=main.metronome))(!(cn=main.sftp)))',
                                ['cn']):
             if not ldap.remove('cn=%s,ou=permission' % per['cn'][0]):
-                raise YunohostError('permission_deletion_failed', permission=permission, app=app)
+                raise YunohostError('permission_deletion_failed',
+                                    permission=per['cn'][0].split('.')[0],
```

Comment:
We won't see any `KeyError` or `IndexError`s here?

Reply:
Normally not. If it happens, we have really a big problem...